### ISSUE TITLE
Consider increasing contrast on Learn to Validate page

### ISSUE THREAD

**Issue by thayeray** [MissingMaps/missingmaps.github.io#439]
Just a comment on the Learn to Validate page: https://www.missingmaps.org/validate/

I am wondering if you might consider increasing the contrast between the text and background in the tutorial. The W3 Consortium:
https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html
recommends a contrast ratio of at least 4.5:1, by my measurements, using the methods described here:
https://www.w3.org/WAI/WCAG21/Techniques/general/G18
the tutorial text is only 3.39:1, making it quite difficult to read.

Thank you for your consideration.

-Thayer

**Comment by danbjoseph** [comment]
this should addressed in #444 
let us know if still an issue

**Comment by thayeray** [comment]
Thanks that looks good.
-Thayer

    On Thursday, April 29, 2021, 02:37:10 PM EDT, Dan Joseph ***@***.***> wrote:  
 
 


this should addressed in #444
let us know if still an issue
